make/Images.gmk

```diff
@@ -117,6 +121,7 @@ $(eval $(call SetupExecute, jlink_jre, \
     OUTPUT_DIR := $(JRE_IMAGE_DIR), \
     SUPPORT_DIR := $(JRE_IMAGE_SUPPORT_DIR), \
     PRE_COMMAND := $(RM) -r $(JRE_IMAGE_DIR), \
+    POST_COMMAND := [ -d $(JRE_IMAGE_DIR)/lib/C ] && $(CP) -rfv $(IMAGES_OUTPUTDIR)/../support/modules_libs/openjceplus/C $(JRE_IMAGE_DIR)/lib, \
```

Comment:
First, I don't think it's reasonable that openjceplus content is sensitive to the effects of `--strip-debug`, and this doesn't address the problem that will continue to exist with users using `jlink --strip-debug` to create their own runtimes. Having said that, even as an _interim fix_, this `POST_COMMAND` needs to change:
* in builds without openjceplus it will just fail (it should do nothing)
* in builds with openjceplus, the action should not be predicated on the existence of the `lib/C` directory (it should fail if it does not exist)
* the command for Windows will probably need to be different
* it should use `$(SUPPORT_OUTPUTDIR)` instead of `$(IMAGES_OUTPUTDIR)/../support`
* the `-v` option should be omitted from the copy command

I suggest that before the call to `SetupExecute` we want something like
```
ifeq ($(BUILD_OPENJCEPLUS), false)
  LEGACY_JRE_POST_COMMAND :=
else ifeq ($(call isTargetOs, windows), true)
  LEGACY_JRE_POST_COMMAND := $(CP) -rf $(SUPPORT_OUTPUTDIR)/modules_cmds/openjceplus/C $(JRE_IMAGE_DIR)/bin
else
  LEGACY_JRE_POST_COMMAND := $(CP) -rf $(SUPPORT_OUTPUTDIR)/modules_libs/openjceplus/C $(JRE_IMAGE_DIR)/lib
endif
```
then the `POST_COMMAND` parameter becomes
```
    POST_COMMAND := $(LEGACY_JRE_POST_COMMAND), \
```

Reply:
Yes agreed ideally `openjceplus` FIPS binaries are not sensitive to the effects of `--strip-debug` or the `strip` command. Users performing `jlink` themselves could run into this same issue. 

This being said we are unsure how to make any recommendations to the OCKC team such that the binary is not sensitive to `strip`. Currently only a single byte is changed by this command so there are no obvious causes here to me ( such as compiling with debug symbols ) that are causing the file to be sensitive to `strip`. Additionally at this point in time the binary has been approved by NIST and is not subject to change for the forseeable future. As an interim fix we certainly need to document in the FIPS related documenation that this argument sent to the jlink command is not supported if somebody were to run this on their own.

I did make updates as recommended in the latest version of the code.
- The suggested variable `LEGACY_JRE_POST_COMMAND` is set only when openjceplus bundling is set. 
- The directory for windows indeed is the bin directory instead of lib. This was modified as suggested.
- Variable `$(SUPPORT_OUTPUTDIR)` was used.
- The verbose `-v` flag was removed from the copy.

Reply:
Hi @johnpeck-us-ibm can you please review the above issue we are trying to fix? We would like to have the ability for the FIPS module to be unaffected by the `jlink` command using the `--strip-debug` argument. This seems to be observed to behaving similar to running a `strip` command within Linux. Currently running this command causes a few bytes to change, which then causes the self verification process of the module to fail upon loading the FIPS certified library.

Reply:
The FIPS code is tamper resistant. 

Any modification to the library will result in the Selftests that FIPS 140 requires to fail. This in turn causes the initialization of the Java Provider to fail. This is done so that the code can not be modified in any way. The ICC libraries already have all the symbols removed. So, the use of --strip-debug is not needed. Also, since the jlink or strip commands rewrites the ICC library the tamper resistance of the library detects this and therefore causes the initialization of library to fail.

Any modification to the library SHOULD cause this failure. Since, this tool is trying to modify the library the tamper resistance is working as designed.

We  CANNOT try and allow ANY kind of modification of the FIPS libraries. This would break module security and mean that the module is not secure.